File: WebAPI/IAI.Models/Models/Interviewer/Requests/InterviewerDashboardRequestParameters.cs
using IAI.Models.Models.Responses;

namespace IAI.Models.Models.Interviewer.Requests
{
    public class InterviewerDashboardRequestParameters : SearchAndPagingParameters
    {
        public Guid InterviewerId { get; set; }
        public string StartDate { get; set; } 
        public string EndDate { get; set; }
        public string? CandidateName { get; set; }
        public int? InterviewTypeId { get; set; }
        public int? StatusId { get; set; }
        public InterviewerDashboardRequestParameters()
        {
        }
    }
}

File: WebApp/iai-app/src/models/interviewer/request/InterviewerDashboardRequestParameters.ts
import { SortDirection } from "../../enums/SortDirectionEnum";

interface InterviewerDashboardRequestParameters {
    page: number,
    pageSize: number,
    sort: string,
    sortDir: SortDirection,
    interviewerId: string,
    startDate: string,
    endDate: string,
    candidateName: string,
    interviewTypeId: number,
    statusId: number
}
export default InterviewerDashboardRequestParameters;